ecms_base/themes/custom/ecms/templates/content/node--event--full.html.twig

```diff
@@ -96,11 +100,30 @@
   {% set allday = TRUE %}
 {% endif %}
 
+
+% set img = '' %}
+  {% if paragraph.field_gallery_item is not empty %}
+    {% if paragraph.field_gallery_item.entity.hasTranslation(langcode) %}
+      {% set img_uri = paragraph.field_gallery_item.entity.translation(langcode).field_media_item_image.entity.getFileUri() %}
+      {% set img_alt = paragraph.field_gallery_item.entity.translation(langcode).field_media_item_image.alt %}
+    {% else %}
+      {% set img_uri = paragraph.field_gallery_item.entity.translation('en').field_media_item_image.entity.getFileUri() %}
+      {% set img_alt = paragraph.field_gallery_item.entity.translation('en').field_media_item_image.alt %}
+    {% endif %}
+    {# TODO: this fails if an uri exists in DB but not on local file system. Need to find a way to check if file exists locally #}
+    {% set img = drupal_image(img_uri, 'wide', {alt: img_alt}, true) %}
+  {% endif %}
```

Comment:
This looks like the code from the paragraphs file, don't think it should be here.